### ISSUE TITLE
"Invalid locator strategy: css selector" while finding by ID

### ISSUE THREAD

**Issue by rseek** [aerokube/ggr#89]
I caught this error on 03.07.2017, trying to find element by ID. (before 03.07.2017, such locators worked fine)

on appium-java-client 4.1.2 (selenium 2.53.1)
```
org.openqa.selenium.UnsupportedCommandException: Invalid locator strategy: css selector (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 107 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'None', ip: '77.88.20.23', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.4', java.version: '1.8.0_121'
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities [{sessionId=26813b028a4e588d6aec711b80bb9551642e773a-6129-4550-b119-ca2c8332a818, platform=ANY}]
Session ID: 26813b028a4e588d6aec711b80bb9551642e773a-6129-4550-b119-ca2c8332a818
*** Element info: {Using=css selector, value=#sign_in}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:40)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:56)
	at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.findElement(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:492)
	at io.appium.java_client.DefaultGenericMobileDriver.findElementByCssSelector(DefaultGenericMobileDriver.java:123)
	at io.appium.java_client.AppiumDriver.findElementByCssSelector(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.findElementByCssSelector(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:415)
	at io.appium.java_client.DefaultGenericMobileDriver.findElementById(DefaultGenericMobileDriver.java:64)
	at io.appium.java_client.AppiumDriver.findElementById(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.findElementById(AndroidDriver.java:1)
	at org.openqa.selenium.By$ById.findElement(By.java:218)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:52)
	at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.findElement(AndroidDriver.java:1)
	at ru.yandex.music.e2etests.common.internal.wrapper.YMobileDriver.findElement(YMobileDriver.java:62)
	at ru.yandex.music.e2etests.common.internal.wrapper.YElement.appiumElement(YElement.java:252)
...
```

**Comment by vania-pooh** [comment]
This is reproducing with release 1.2.3.

**Comment by vania-pooh** [comment]
But all the stuff works with 1.2.2.

**Comment by vania-pooh** [comment]
Fixed in #95.
